public/app/plugins/datasource/prometheus/datasource.test.ts

```diff
@@ -1601,9 +1601,24 @@ describe('PrometheusDatasource', () => {
           text: expectedRangeSecond * 1000,
           value: expectedRangeSecond * 1000,
         },
+        __rate_interval: {
+          text: '75s',
+          value: '75s',
+        },
       });
     });
   });
+
+  describe('The __rate_interval variable', () => {
+    it('should be 4 times the scrape interval if interval + scrape interval is lower', () => {
+      const { __rate_interval } = ds.getRateIntervalScopedVariable(15, 15);
+      expect(__rate_interval.value).toBe('60s');
+    });
+    it('should fall back to 15s if interval is 0', () => {
```

Comment:
s/15s/60s/

Reply:
I'm not sure what you mean by that.

Reply:
Perhaps I misread the test, but it appears the whole sentence is "The __rate_interval variable should fall back to 15s if interval is 0" but the __rate_interval is in fact 60s, not 15s, see line 1619.

Reply:
I see, you are right.